### PR TITLE
Included new paid article writing opportunity at Enlear https://www.enlear.com/

### PR DESCRIPTION
**Pull request by indrajithek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -195,6 +195,10 @@ A list of companies that have paid Developer Community Writer Programs.
 
 - [WPHUB](https://www.wphub.com/write-for-us/) - $200 per piece
   > Wordpress tutorials and articles.
+
+- [Enlear](https://www.enlear.com/) - Vary per writer experiance
```
Comment (malgamves) [inline README.md:199]: there's  a typo on experience.. also could you post the ranges for beginner and advanced writers?

**Review by malgamves — CHANGES_REQUESTED** [review]

**Review by indrajithek — COMMENTED** [review]
updated

File: README.md [hunk 2]
```diff
@@ -195,6 +195,10 @@ A list of companies that have paid Developer Community Writer Programs.
 
 - [WPHUB](https://www.wphub.com/write-for-us/) - $200 per piece
   > Wordpress tutorials and articles.
+
+- [Enlear](https://www.enlear.com/) - $50 to $150 per article
+  > Get paid to write about broad area of technical topics. Including Vue, Angular, React, Node, JS, DynamoDB
+
```
Comment (Qodestackr) [inline README.md:201]: I opened an account here but my Enlear account is saying free. Do I need to contribute first before it becomes a paid status?

Comment (indrajithek) [reply README.md:201]: yes

Comment (Qodestackr) [reply README.md:201]: Okay thanks

Comment (malgamves) [reply README.md:201]: thanks for adding the range!

**Review by malgamves — APPROVED** [review]

**Comment by malgamves** [comment]
Thanks for adding this!

**Pull request merged**